src/Options/index.js

```diff
@@ -1,11 +1,11 @@
 // @flow
 import { AnalyticsAdapter } from '../Adapters/Analytics/AnalyticsAdapter';
-import { FilesAdapter } from '../Adapters/Files/FilesAdapter';
```

Comment:
Could you revert this reordering of import statements? We want to keep only PR changes that are necessary for the PR itself.